<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="GUO-PC" name="com.gorky.appiumdemo.testcase.home.HomePage_001_HomeCase_Test" tests="1" failures="1" timestamp="29 Nov 2017 03:28:49 GMT" time="0.000" errors="0">
  <testcase name="homeDemo" time="0.000" classname="com.gorky.appiumdemo.testcase.home.HomePage_001_HomeCase_Test">
    <failure type="java.lang.RuntimeException" message="java.lang.AssertionError: 不能读取文件： [data/home.xls]">
      <![CDATA[java.lang.RuntimeException: java.lang.AssertionError: 不能读取文件： [data/home.xls]
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.AssertionError: 不能读取文件： [data/home.xls]
	at org.testng.Assert.fail(Assert.java:94)
	at com.gorky.appiumdemo.utils.ExcelDataProvider.<init>(ExcelDataProvider.java:68)
	at com.gorky.appiumdemo.base.BasePrepare.dataFortestMethod(BasePrepare.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:153)
	... 16 more
]]>
    </failure>
  </testcase> <!-- homeDemo -->
</testsuite> <!-- com.gorky.appiumdemo.testcase.home.HomePage_001_HomeCase_Test -->
